moveit_experimental/moveit_jog_arm/test/arm_setup/config/jog_settings.yaml

```diff
@@ -1,33 +1,55 @@
+###############################################
+# Modify all parameters related to jogging here
+###############################################
 use_gazebo: false # Whether the robot is started in a Gazebo simulation environment
-check_collisions: true # Check collisions?
+
+## Properties of incoming commands
+robot_link_command_frame:  base_link  # commands must be given in the frame of a robot link. Usually either the base or end effector
 command_in_type: "unitless" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
-scale: # Only used if command_in_type=="unitless"
-  linear:  0.003  # Max linear velocity. Meters per publish_period.
-  rotational:  0.006  # Max angular velocity. Rads per publish_period.
-  joint: 0.01  # Max joint angular/linear velocity. Rads or Meters per publish period.
-cartesian_command_in_topic:  jog_server/delta_jog_cmds # Topic for xyz commands
-joint_command_in_topic: jog_server/joint_delta_jog_cmds # Topic for angle commands
-command_frame:  base_link  # TF frame that incoming cmds are given in
-incoming_command_timeout:  1  # Stop jogging if X seconds elapsed without a new cmd
-joint_topic:  joint_states
-move_group_name:  panda_arm  # Often 'manipulator' or 'arm'
-lower_singularity_threshold:  30  # Start decelerating when the condition number hits this (close to singularity). Larger --> closer to singularity
-hard_stop_singularity_threshold: 45 # Stop when the condition number hits this. Larger --> closer to singularity
-collision_proximity_threshold: 0.03 # Start decelerating when a collision is this far [m]
-planning_frame: base_link  # The MoveIt planning frame. Often 'base_link'
-low_pass_filter_coeff: 2.  # Larger-> more smoothing to jog commands, but more lag.
+scale:
+  linear:  0.003  # Max linear velocity. Meters per publish_period. Units is [m/s]. Only used if command_in_type=="unitless"
```

Comment:
1. I found out recently that the joint velocity cap is still used, regardless of command_in_type. That makes sense, because the joints will be moving either way. So that comment only applies to the linear and rotational Cartesian commands.

2. I don't think so. If speed_units, no scaling is applied... it's assumed the user has already scaled the commands how they want.